core/ir/instr_shared.c

```diff
@@ -1490,15 +1490,15 @@ instr_decode_cti(dcontext_t *dcontext, instr_t *instr)
      */
     if (!instr_opcode_valid(instr) ||
         (instr_is_cti(instr) && !instr_operands_valid(instr))) {
-        byte *next_pc;
+        DEBUG_EXT_DECLARE(byte * next_pc;)
```

Comment:
DEBUG_EXT_DECLARE doesn't seem to do anything that's "declaration" related. Why not just IF_DEBUG/DODEBUG?

Reply:
I thought DEBUG_EXT_DECLARE was for CLIENT_ASSERT which is subtly different in which places it's enabled vs DEBUG -- at least that's what the comment says:
```C
/* For use in CLIENT_ASSERT or elsewhere that exists even if
 * STANDALONE_DECODER is defined.
 */
```

Reply:
Sticking with what is intended for use in this situation.